tests/unit_tests/test_ecephys.py

```diff
@@ -215,3 +216,31 @@ def test_check_spike_times_not_in_unobserved_interval_multiple_units():
         object_name="TestUnits",
         location="/",
     )
+
+
+def test_check_ascending_spike_times_pass():
+    units_table = Units()
+    units_table.add_unit(spike_times=[0.0, 0.1])
+    units_table.add_unit(spike_times=[1.0, 2.0])
+    assert check_ascending_spike_times(units_table=units_table) is None
+
+
+def test_check_ascending_spike_times_w_missing_spike_times_pass():
+    units_table = Units()
+    units_table.add_unit(spike_times=[0.0, 0.1])
+    units_table.add_unit(spike_times=[])
```

Comment:
```suggestion
def test_check_ascending_spike_times_missing_column_skip():
    units_table = Units()
```

The point of this test is to evoke the early return of https://github.com/NeurodataWithoutBorders/nwbinspector/pull/405/files#diff-dfe54828956c62fba202acb9497ca060f0448c8e0cd1b945cf31d0daaa53d769R87

which is for the case where we have an empty Units table

This would of course trigger a separate check function: https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/src/nwbinspector/checks/tables.py#L40

but the point is still that we don't want the logic of the check to reach https://github.com/NeurodataWithoutBorders/nwbinspector/pull/405/files#diff-dfe54828956c62fba202acb9497ca060f0448c8e0cd1b945cf31d0daaa53d769R89 and error out because

Reply:
@CodyCBakerPhD yeah but I think it would also not be a bad idea to include the existing test as well

Reply:
To check the safe range slicing of https://github.com/NeurodataWithoutBorders/nwbinspector/pull/405/files#diff-dfe54828956c62fba202acb9497ca060f0448c8e0cd1b945cf31d0daaa53d769R90, sure

Both would be fine to have in that case

Reply:
Ok, keeping both